src/Datadog.Trace.ServiceFabric/Remoting.cs

```diff
@@ -216,14 +216,14 @@ private static void GetMessageHeaders(EventArgs? eventArgs, out ServiceRemotingR
 
         private static void InjectContext(PropagationContext context, IServiceRemotingRequestMessageHeader messageHeaders)
         {
-            if (context.TraceId == 0 || context.ParentSpanId == 0)
+            if (context.TraceId.Equals(TraceId.Zero) || context.ParentSpanId == 0)
```

Comment:
I think it would be more idiomatic if we overload the operators for `!=` or `==`:

```
context.TraceId == TraceId.Zero`
```

I was thinking of adding a property to `TraceId` named `Assigned` but I think it would just add an unnecessary layer of indirection

Reply:
I like the == idea, but would like to hear others' opinion on that first :)

Reply:
I like the `==` idea. I also thought about having a property or method similar to `Assigned`, maybe even `IsZero`, but in the end thought it was unnecessary too.

Reply:
Changed to ==